package.json

```diff
@@ -46,7 +46,6 @@
     "git-rev": "git://github.com/bewest/git-rev.git"
   },
   "devDependencies": {
-    "coveralls": "~2.11.2",
```

Comment:
Why remove the Dev dependency for coveralls and replace it with a `npm install` in `after_script`?